src/cli-constant.js

```diff
@@ -82,7 +82,6 @@ const detailedOptions = normalizeDetailedOptions({
     category: CATEGORY_FORMAT,
     forwardToApi: true,
     description: "Include parentheses around a sole arrow function parameter.",
-    default: "avoid",
```

Comment:
Are you going to fill this back in once we've decided on the new default?

Reply:
Oops, forgot to mention why I removed this line: we use the default value from `options.js` for all format options.

https://github.com/prettier/prettier/blob/fbbfa525171f36632ae879010cba92af9a5fd896/src/cli-util.js#L561-L579

Reply:
Hmm, that's feels a bit unintuitive, but changing it is out of the scope of this PR